packages/components/src/grid/grid-utils.js

```diff
@@ -0,0 +1,19 @@
+const ALIGNMENTS = {
+	bottom: { alignItems: 'flex-end', justifyContent: 'center' },
+	bottomLeft: { alignItems: 'flex-start', justifyContent: 'flex-end' },
+	bottomRight: { alignItems: 'flex-end', justifyContent: 'flex-end' },
+	center: { alignItems: 'center', justifyContent: 'center' },
+	edge: { alignItems: 'center', justifyContent: 'space-between' },
```

Comment:
This one reads a little odd to me as `edge`. Wouldn't something like `spaced` be more clear?

Reply:
Hmm! That's a good suggestion :D.
I remember `edge` was a value I was struggling with at the time of naming.
I like `spaced`  a lot more.

Reply:
Sweet!